GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -88,6 +89,9 @@ public bool TryGetNormalizedPathRoot(string path, out string pathRoot, out strin
 
         public class GVFSPlatformConstants
         {
+            public const string WindowsPlatformName = "Windows";
+            public const string MacPlatformName = "macOS";
```

Comment:
Can you move these into `WindowsPlatform` and `MacPlatform`. 

`GVFSPlatform` should not know/care which platforms implement it.

(If you want the property to live in `GVFSPlatformConstants` you could take the same approach used for `installerExtension` and have the value passed in via the `GVFSPlatform` constructor)